tardis/montecarlo/montecarlo_numba/interaction.py

```diff
@@ -56,10 +54,13 @@ def thomson_scatter(r_packet, time_explosion):
 def line_scatter(r_packet, time_explosion, line_interaction_type, numba_plasma):
     """
     Line scatter function that handles the scattering itself, including new angle drawn, and calculating nu out using macro atom
-    r_packet: RPacket
-    time_explosion: float
-    line_interaction_type: enum
-    numba_plasma: NumbaPlasma
+    
+    Parameters
+    ----------
+    r_packet : RPacket
```

Comment:
Like @wkerzendorf suggested - you should write the absolute class: `tardis.montecarlo.montecarlo_numba.r_packet.RPacket`